src/views/BioPage.jsx

```diff
@@ -1,8 +1,92 @@
 // LIBRARY IMPORTS
 import React from 'react';
-
+import { useParams } from 'react-router-dom';
 // LOCAL IMPORTS
+import { bioData } from '../assets/bioData.js';
+import SingleCard from '../components/SingleCard.jsx';
+import ResourceCard from '../components/ResourceCard.jsx';
+
+export const BioPage = () => {
+  const { id } = useParams();
+  const currentIndex = bioData.findIndex(item => item.id === parseInt(id, 10));
+
+  const nextCards = bioData
+    .filter(
+      (item, index) =>
+        index !== currentIndex &&
+        index > currentIndex &&
+        index < currentIndex + 4,
+    )
+    .map(item => (
+      <SingleCard
+        key={item.id}
+        id={item.id}
+        image={item.image}
+        name={item.fullName}
+        body={item.occupation}
+      />
+    ));
```

Comment:
I added this `nextCards` function which displays the next cards in bioData by id. I'm not sure if this'll do or would it be better to make a function that displays _random_ cards?

Reply:
This is great for MVP - if we have time to update I think we should make it random